src/openms/source/ANALYSIS/ID/FalseDiscoveryRate.cpp

```diff
@@ -1570,7 +1653,7 @@ namespace OpenMS
     for (size_t j = 0; j < scores_labels.size(); ++j)
     {
       sum += scores_labels[j].first;
-      estimatedFDR[j] = sum / (j+1.0);
+      estimatedFDR[j] = sum / (double(j)+1.0);
```

Comment:
size_t is guaranteed to be converted to double here. cast not needed

Reply:
Yes it's guaranteed but you will get conversion warnings if you do not explicitly cast